### PR TITLE
feat(#1637): replace sanchonet in copy with voltaire

### PR DESCRIPTION
**Pull request by MSzalowski**
## List of changes

- Replace sanchonet with voltaire
- Consume network from get /network/metrics in order to make the GovTool network agnostic
- Add dynamic link to cexplorer based on the network

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/1637)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by MSzalowski** [comment]
@m-i-k-e Could you have a look on https://dev-sanchonet.govtool.byron.network/ and leave approval once the unsancho version of GovTool looks as expected?

File: govtool/metadata-validation/src/main.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ async function bootstrap() {
       exceptionFactory: () => ({ status: 200, valid: false }),
     }),
   );
+  console.log({ PORT: process.env.PORT });
```
Comment (jdyczka) [inline govtool/metadata-validation/src/main.ts:28]: Is this `console.log` intentional?

**Review by jdyczka — APPROVED** [review]

File: govtool/metadata-validation/src/main.ts (hunk 1 above)
Comment (MSzalowski) [reply govtool/metadata-validation/src/main.ts]: Thank you for spotting this. I removed that :saluting_face:

**Comment by m-i-k-e** [comment]
looks good

**Pull request merged**